packages/frameworks/index.d.ts

```diff
@@ -13,12 +13,12 @@ export type Setting = SettingValue | SettingPlaceholder;
 
 export interface Framework {
   name: string;
-  slug: string;
+  slug: string | null;
   logo: string;
-  demo: string;
-  tagline: string;
-  website: string;
-  description: string;
+  demo?: string;
+  tagline?: string;
+  website?: string;
+  description?: string;
```

Comment:
Perhaps there should be a description for "Other" to minimize the number of optional properties here and give a longer description to the user?

Reply:
Added `description`, the others will stay optional